plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1570,144 +1610,186 @@ SecRule REQUEST_FILENAME "@contains /settings/api/admin/twofactorauth" \
+    ctl:ruleRemoveTargetById=942432;ARGS:configValue,\
+    ctl:ruleRemoveTargetById=942340;ARGS:configValue,\
+    ctl:ruleRemoveTargetById=942370;ARGS:configValue,\
+    ctl:ruleRemoveTargetById=932240;ARGS:json.configValue,\
+    ctl:ruleRemoveTargetById=942200;ARGS:json.configValue,\
+    ctl:ruleRemoveTargetById=942430;ARGS:json.configValue,\
+    ctl:ruleRemoveTargetById=942431;ARGS:json.configValue,\
+    ctl:ruleRemoveTargetById=942432;ARGS:json.configValue,\
+    ctl:ruleRemoveTargetById=942340;ARGS:json.configValue,\
+    ctl:ruleRemoveTargetById=942370;ARGS:json.configValue"
 
-# Enabling any default theme requires the PUT method.
-SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/theming/api/v[0-9]+/theme/.*?/enable" \
-    "id:9508604,\
+# Validating password against password policy
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/apps/password_policy/api/v[0-9]/validate$" \
+    "id:9508615,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
-    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:password,\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.password"
 
-# When trying to enable keyboard shortcuts
-SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/provisioning_api/api/v[0-9]+/config/users/theming/shortcuts_disabled" \
-    "id:9508605,\
+# Personal Settings --> Security --> Two-Factor Authentication
+# Disabling email 2FA
+SecRule REQUEST_FILENAME "@endsWith /apps/twofactor_email/settings/disable" \
+    "id:9508616,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
 
-# Disabling background
-SecRule REQUEST_FILENAME "@endsWith /apps/theming/background/custom" \
+# Personal Settings --> Personal Info --> Profile Visibility
+# Changing profile visibility settings
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/profile/[^/]+$" \
     "id:9508617,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
-    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
+
+# Personal Settings --> Personal Info --> Website
+# When adding a link to your own user profile
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/cloud/users/$" \
+    "id:9508618,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
+    ctl:ruleRemoveTargetById=931130;ARGS:value,\
+    ctl:ruleRemoveTargetById=931130;ARGS:json.value"
 
-# When disabling settings under Settings --> Availability
-SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/apps/provisioning_api/api/v[0-9]/config/users/dav/user_status_automation$" \
-    "id:9508619,\
+#
+# [ Administration settings ]
+#
+
+# Personal Settings --> Privacy --> Who has access to your data?
```

Comment:
This one should go under personal settings, right?

Reply:
This is shown in personal settings within Nextcloud's UI, but only admins are able to add/remove users to "who can see your data"